src/components/MovieCard/MovieCard.js

```diff
@@ -47,11 +47,11 @@ const MovieCard = ({ movie }) => {
         <></>
       )}
       <div className="movie-card">
-        <div className="movie-card__container ">
+        <div className="movie-container ">
           <Icon
             path={mdiDotsVerticalCircle}
             size={"36px"}
-            className="menu-icon"
+            className="movie-container__icon"
```

Comment:
Honestly, this class makes more sense when it follows the main container syntax, please update other references.

```suggestion
            className="movie-card__icon"
```